plugins/resizimg/trumbowyg.resizimg.js

```diff
@@ -191,25 +193,26 @@
         };
     };
 
-    // object to interact with canvas
-    var resizeWithCanvas = new ResizeWithCanvas();
-
-    function destroyResizable(trumbowyg) {
-        // clean html code
-        trumbowyg.$ed.find('canvas.resizable')
-            .resizable('destroy')
-            .off('mousedown', preventDefault)
-            .removeClass('resizable');
-
-        resizeWithCanvas.reset();
-
-        trumbowyg.syncCode();
-    }
```

Comment:
What's the point of moving that function into the resizimg init function?

Reply:
Hi @Alex-D , I have modified the constructor to pass an instance of trumbowyg to fix an issue that occour when you start resize an image, press canc or del to delete the image and save your text. In the previous code, if you do this actions, the image is not removed from the html code in trumbowyg and you can see it again on the next edit.

I have moved the destroyResizable function too because it uses the instance of ResizeWithCanvas inside it.